libminifi/include/utils/Monitors.h

```diff
@@ -41,7 +41,7 @@ class AfterExecute {
 
   AfterExecute() = default;
 
-  explicit AfterExecute(AfterExecute &&other) {
+  explicit AfterExecute(AfterExecute&& /*other*/) {
```

Comment:
Can we remove `explicit` and explicitly `default` this?

Reply:
Yes, updated in 267cda83a77a5a6d05f9df714d0c923814351826